src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -60,6 +60,10 @@ public async Task<(Error, GitHubUser)> GetUserByLogin(string login)
                 _rateLimitError = Errors.GitHubApiFailed(apiDetail, ex);
                 return (_rateLimitError, null);
             }
+            catch (AbuseException ex)
+            {
+                return (Errors.GitHubApiFailed(apiDetail, ex, ex.RetryAfterSeconds), null);
```

Comment:
can we combine these two api exception handler together? seems they are almost same